code/modules/telesci/bscrystal.dm

```diff
@@ -25,7 +32,10 @@
 
 /obj/item/bluespace_crystal/throw_impact(atom/hit_atom)
 	..()
-	if(isliving(hit_atom))
+	var/turf/T = get_turf(src)
```

Comment:
You might want to compress this into a get_z_level() proc or something, might be helpful.